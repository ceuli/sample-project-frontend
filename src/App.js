import React from 'react'

import NavBar from '@components/navbar'

const App = () => {
	return (
		<div>
			<h2>App</h2>
			<NavBar />
		</div>
	)
}

export default App
