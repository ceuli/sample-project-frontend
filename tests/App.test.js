import React from 'react'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import App from '../src/App'

describe('<App/>', () => {
	it('should render without crashing', () => {
		const wrapper = shallow(<App />)

		expect(toJson(wrapper)).toMatchSnapshot()
	})
})
