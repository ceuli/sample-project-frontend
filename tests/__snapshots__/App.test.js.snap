// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should render without crashing 1`] = `
<div>
  <h2>
    App
  </h2>
  <NavBar />
</div>
`;
